en/docs/reference/connectors/email-connector/email-connector-example.md

```diff
@@ -18,22 +18,23 @@ The example consists of an API named as EmailConnector API with two resources `s
 
 If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
 
-## Configure the connector in WSO2 Integration Studio
+## Setup the Integration Project
```

Comment:
```suggestion
## Setting up the Integration Project
```

Let's use this format to ensure consistency of the titles across the page.

Reply:
Sure